libbeat/docs/config-file-format.asciidoc

```diff
@@ -0,0 +1,355 @@
+[[config-file-format]]
+== Config file format
+
+Beats config files are based on http://www.yaml.org[YAML], a file format that is
+easier to read and write then other common data formats like XML or JSON.
+
+In beats all YAML files start with a "dictionary" (often called a "hash"). In
+addition to dictionaries, YAML also supports lists, numbers, strings, and many
+other data types. All members of lists and dictionaries must have the same
```

Comment:
This is grammatically ambiguous. Might be better to say, "All members of the same list or dictionary must have the same indentation level."